README.md

```diff
@@ -258,6 +258,32 @@ Use the `limit` option to alter the display width. Example:
 }
 ```
 
+### Default Clause/Constant
+Use the `default` property to have MySQL handle setting column `DEFAULT` value.
+```javascript
+"status": {
+  "type": "string",
+  "mysql": {
+    "default": "pending"
+  }
+},
+"number": {
+  "type": "number",
+  "mysql": {
+    "default": 256
+  }
+}
+```
+For the date or timestamp types use `CURRENT_TIMESTAMP` or `now`:
+```javascript
+"last_modified": {
+  "type": "date",
+  "mysql": {
+    "default":"CURRENT_TIMESTAMP"
+  }
+}
+```
```

Comment:
Added in this commit: https://github.com/strongloop/loopback-connector-mysql/pull/319/commits/97009acc18a75ee02243e4c9d159a8e5a9a35dd9